src/SFA.DAS.EmployerAccounts/Interfaces/IClientContentService.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Threading.Tasks;
+
+namespace SFA.DAS.EmployerAccounts.Interfaces
+{
+    public interface IClientContentService
+    {
+        Task<string> GetContent(string type, string clientId);
```

Comment:
Can we name this method Get rather than GetContent.  It's the client content service so we would expect it to return client content.